packages/techdocs-common/src/stages/publish/awsS3.ts

```diff
@@ -149,6 +130,35 @@ export class AwsS3Publish implements PublisherBase {
     this.logger = logger;
   }
 
+  /**
+   * Check if the defined bucket exists. Being able to connect means the configuration is good
+   * and the storage client will work.
+   */
+  async validateConfiguration(): Promise<ConfigurationValidationResponse> {
+    try {
+      await this.storageClient
+        .headBucket({ Bucket: this.bucketName })
+        .promise();
+
+      this.logger.info(
+        `Successfully connected to the AWS S3 bucket ${this.bucketName}.`,
```

Comment:
Now that we may not necessarily _only_ be running this at start-up, it may make sense to tweak these success messages a little bit to be more of a representation of continuous state?  e.g. instead of `Successfully connected to...`, something like `Able to connect to...`

Reply:
Tbh, I don't really like emitting logs for the happy-case at all. Sure, they are nice to have for a one-off call, but if this might be used to be run repeatedly as a healthcheck, it is not the task of the logging tool to monitor health checks, but there will be a library/external tool that calls it and should do the logging if needed. I would prefer to leave it as-is for now.